config/application.rb

```diff
@@ -109,7 +109,8 @@ class Application < Rails::Application
 
     config.autoload_once_paths << Rails.root.join("lib", "vmdb", "console_methods.rb").to_s
 
-    config.middleware.use 'RequestStartedOnMiddleware'
+    require_relative '../lib/request_started_on_middleware'
+    config.middleware.use RequestStartedOnMiddleware
```

Comment:
Could we also consider putting this change in the initializer as well, ala something like:

https://github.com/rails/rails/commit/83b767ce#commitcomment-19818100

I have no preference if you do, it would just avoid a require when it will probably get autoloaded as part of app initialization anyway.